AsyncDisplayKit/Private/ASDisplayNode+UIViewBridge.mm

```diff
@@ -143,22 +143,7 @@ - (void)setFrame:(CGRect)rect
   ASDisplayNodeAssert(CATransform3DIsIdentity(self.transform), @"-[ASDisplayNode setFrame:] - self.transform must be identity in order to set the frame property.  (From Apple's UIView documentation: If the transform property is not the identity transform, the value of this property is undefined and therefore should be ignored.)");
```

Comment:
Looks great!  Thanks for keeping the assertion here — this is not technically the behavior of setFrame: for UIView so I don't want to enable people to call setFrame: with a transform set.